addons/mrp/models/mrp_workorder.py

```diff
@@ -4,6 +4,7 @@
 from datetime import datetime
 from dateutil.relativedelta import relativedelta
 from collections import defaultdict
+import math
```

Comment:
Import only `floor` (don't forget to change it's call in the onchange below).
``` python3
from math import floor
```

Reply:
@svs-odoo Changes done.